scss/_tables.scss

```diff
@@ -48,6 +48,18 @@
   > tbody {
     vertical-align: inherit;
   }
+
+  // Boosted mod
+  // When using checkboxes in the first column, force width of this column and add negative right margin
+  // to checkbox, to ensure correct spacing on the first column.
+  &.has-checkbox tr > :first-child {
+    width: $spacer * 2;
+
+    .form-check {
```

Comment:
The whole rule on `.form-check` could be replaced by `max-width: $spacer * 2;`. Not sure what's the best here 🤷

Reply:
Can someone try it? If we can do the same thing with a reference to `$spacer`, it would be better than a negative margin and magical value.

Reply:
Ok it works with both width and max-width... 
Tested on Windows/Chrome, Windows/Firefox and on Browserstack : iOS/Safari and MacOS/Safari